### PR TITLE
JNG-4308 deep copy test

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4308" title="JNG-4308" target="_blank">TaskJNG-4308</a>  Deep copy test
  </td></table>
  <br />

### CODE REVIEW

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java [hunk 1]
```diff
@@ -273,4 +275,18 @@ void testAbstractDAOOperations() {
...
     }
+
+    @Test
+    @Disabled
+    void testDeepCopyConstructor() {
+        //When we add a composition Entity we must copy it, because that comp entity belong the created entity
+
+        //TODO-JNG-4317
+
+        assertNotEquals(entityA.getSingleRequiredConA().get__identifier(),singleRequiredConA.get__identifier());
```
Comment (robertcsakany) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java]: Space after , !

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java [hunk 2]
```diff
@@ -273,4 +275,18 @@ void testAbstractDAOOperations() {
 
         assertEquals(Optional.of("B"), updatedB.get().getStringB());
     }
+
+    @Test
+    @Disabled
+    void testDeepCopyConstructor() {
+        //When we add a composition Entity we must copy it, because that comp entity belong the created entity
```
Comment (robertcsakany) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java:282]: Put the model init here from @Berfore

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java (hunk 2 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java:282]: [c26c058](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/106/commits/c26c058eef801889ef8c1ad20c24162841342beb)

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java (hunk 1 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/CompositionRelationshipsTest.java]: [c26c058](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/106/commits/c26c058eef801889ef8c1ad20c24162841342beb)

**Review by blackbelt-oss — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Pull request merged**